### PR TITLE
Move pill to core

### PR DESCRIPTION
**Pull request by Fercas123**

### CODE REVIEW

File: packages/core/stories/pill/pill.doc.stories.mdx [hunk 1]
```diff
@@ -44,7 +43,7 @@ The default Pill is often used for tagging purposes, or to categorize content. P
   categories. Use [Icon Descriptor](/docs/documentation-icons-icon--docs) instead.
 
 <Canvas>
-  <Story id="lab-pill-next--default" />
+  <Story id="lab-pill--default" />
```
Comment (feliciackh) [inline packages/core/stories/pill/pill.doc.stories.mdx:46]:
```suggestion
  <Story id="core-pill--default" />
```

File: packages/core/stories/pill/pill.doc.stories.mdx [hunk 2]
```diff
@@ -53,7 +52,7 @@ Icons can be displayed on the left of the Pill label to visually
 reinforce the theme or grouping of that Pill, and for quick recognition.
 
 <Canvas>
-  <Story id="lab-pill-next--icon" />
+  <Story id="lab-pill--icon" />
```
Comment (feliciackh) [inline packages/core/stories/pill/pill.doc.stories.mdx:55]:
```suggestion
  <Story id="core-pill--icon" />
```

File: packages/core/stories/pill/pill.doc.stories.mdx [hunk 3]
```diff
@@ -87,17 +86,17 @@ ensure it provides an accurate role to screen readers.
 ## API
 
 ```
-import { PillNext } from "@salt-ds/lab";
+import { Pill } from "@salt-ds/lab";
```
Comment (feliciackh) [inline packages/core/stories/pill/pill.doc.stories.mdx:89]:
```suggestion
import { Pill } from "@salt-ds/core";
```

File: site/src/examples/pill/Closable.tsx [hunk 4]
```diff
@@ -1,13 +1,13 @@
 import { ReactElement } from "react";
-import { PillNext } from "@salt-ds/lab";
+import { Pill } from "@salt-ds/lab";
```
Comment (feliciackh) [inline site/src/examples/pill/Closable.tsx:2]:
```suggestion
import { Pill } from "@salt-ds/core";
```

File: site/src/examples/pill/Default.tsx [hunk 5]
```diff
@@ -1,10 +1,10 @@
 import { ReactElement } from "react";
-import { PillNext } from "@salt-ds/lab";
+import { Pill } from "@salt-ds/lab";
```
Comment (feliciackh) [inline site/src/examples/pill/Default.tsx:2]:
```suggestion
import { Pill } from "@salt-ds/core";
```

File: site/src/examples/pill/Disabled.tsx [hunk 6]
```diff
@@ -1,6 +1,4 @@
 import { ReactElement } from "react";
-import { PillNext } from "@salt-ds/lab";
+import { Pill } from "@salt-ds/lab";
```
Comment (feliciackh) [inline site/src/examples/pill/Disabled.tsx:2]:
```suggestion
import { Pill } from "@salt-ds/core";
```

File: site/src/examples/pill/Icon.tsx [hunk 7]
```diff
@@ -1,13 +1,13 @@
 import { ReactElement } from "react";
-import { PillNext } from "@salt-ds/lab";
+import { Pill } from "@salt-ds/lab";
```
Comment (feliciackh) [inline site/src/examples/pill/Icon.tsx:2]:
```suggestion
import { Pill } from "@salt-ds/core";
```

File: site/src/layouts/DetailComponent/SecondarySidebar.tsx [hunk 8]
```diff
@@ -1,6 +1,6 @@
 import React, { FC, ReactNode } from "react";
 import { Image } from "@jpmorganchase/mosaic-site-components";
-import { PillNext } from "@salt-ds/lab";
+import { Pill } from "@salt-ds/lab";
```
Comment (feliciackh) [inline site/src/layouts/DetailComponent/SecondarySidebar.tsx:3]:
```suggestion
import { Pill } from "@salt-ds/core";
```

File: site/src/layouts/DetailComponent/SecondarySidebar.tsx [hunk 9]
```diff
@@ -1,6 +1,6 @@
 import React, { FC, ReactNode } from "react";
 import { Image } from "@jpmorganchase/mosaic-site-components";
-import { Pill } from "@salt-ds/lab";
+import { Pill } from "@salt-ds/core";
 import { Link } from "@salt-ds/core";
```
Comment (feliciackh) [inline site/src/layouts/DetailComponent/SecondarySidebar.tsx:4]: can be combined with line above

**Review by feliciackh — DISMISSED** [review]

File: packages/core/stories/pill/pill.doc.stories.mdx
Comment (alycrys) [inline packages/core/stories/pill/pill.doc.stories.mdx:1]: Should we delete this @joshwooding ?

File: packages/lab/src/__tests__/__e2e__/combo-box-deprecated/ComboBox.accessibility.cy.tsx [hunk 10]
```diff
@@ -151,7 +151,7 @@ describe("A multi-select combo box", () => {
       cy.findByRole("textbox").should("not.have.attr", "aria-activedescendant");
     });
 
-    it("should attach correct aria-activedescendant when navigating through list and pills", () => {
+    it.skip("should attach correct aria-activedescendant when navigating through list and pills", () => {
```
[the remark this answers is not in the document]
Comment (Fercas123) [reply packages/lab/src/__tests__/__e2e__/combo-box-deprecated/ComboBox.accessibility.cy.tsx]: using the new pill for the deprecated components needed some small tweaks, all done

**Review by feliciackh — DISMISSED** [review]

**Review by joshwooding — APPROVED** [review]
The site build is failing due to a ComboBox example:

image

**Review by joshwooding — DISMISSED** [review]
See comment

File: packages/lab/CHANGELOG.md [hunk 11]
```diff
@@ -43,7 +43,7 @@
 
 ### Patch Changes
 
-- 2fd8c03c: Pill Next's `className` prop is now forwarded to its wrapper instead of the nested button.
+- 2fd8c03c: Pill's `className` prop is now forwarded to its wrapper instead of the nested button.
```
Comment (joshwooding) [inline packages/lab/CHANGELOG.md:46]: Historic changelogs shouldn't be changed like this.

**Comment by joshwooding** [comment]
image

**Review by joshwooding — APPROVED** [review]

**Review by bhoppers2008 — APPROVED** [review]

**Pull request merged**